### PR TITLE
Print execution plan information in console

### PR DESCRIPTION
**Pull request by yixinglu**
Refactor some implementations.

for example:

```text

Welcome to Nebula Graph v2.0.0-alpha!

(user@nebula) [(none)]> use nba;

Sat, 01 Aug 2020 11:59:15 CST

(user@nebula) [nba]> explain go 3 steps from "Boris Diaw" over like yield like._dst union go 4 steps from "Tim Duncan" over like yield like._dst
+----+--------------+--------------+-------------------+--------------------------+----------------+-------------------------------+
| id | name         | dependencies | output_var        | branch_info              | profiling_data | description                   |
+----+--------------+--------------+-------------------+--------------------------+----------------+-------------------------------+
| 17 | DataCollect  | 13           | __DataCollect_17  |                          |                | description: DataCollect      |
+----+--------------+--------------+-------------------+--------------------------+----------------+-------------------------------+
| 13 | Dedup        | 12           | __Dedup_13        |                          |                | description: Dedup            |
+----+--------------+--------------+-------------------+--------------------------+----------------+-------------------------------+
| 12 | Union        | 5,11         | __Union_12        |                          |                | description: Union            |
+----+--------------+--------------+-------------------+--------------------------+----------------+-------------------------------+
|  5 | Project      | 4            | __Project_5       |                          |                | description: Project          |
+----+--------------+--------------+-------------------+--------------------------+----------------+-------------------------------+
|  4 | GetNeighbors | 3            | __GetNeighbors_4  |                          |                | description: GetNeighbors     |
+----+--------------+--------------+-------------------+--------------------------+----------------+-------------------------------+
|  3 | Loop         | 14           | __Loop_3          |                          |                | description: Loop             |
+----+--------------+--------------+-------------------+--------------------------+----------------+-------------------------------+
|  2 | Project      | 1            | __UNAMED_VAR_0    | branch: true, node_id: 3 |                | description: Project          |
+----+--------------+--------------+-------------------+--------------------------+----------------+-------------------------------+
|  1 | GetNeighbors | 0            | __GetNeighbors_1  |                          |                | description: GetNeighbors     |
+----+--------------+--------------+-------------------+--------------------------+----------------+-------------------------------+
|  0 | Start        |              | __Start_0         |                          |                | description: Start            |
+----+--------------+--------------+-------------------+--------------------------+----------------+-------------------------------+
| 14 | MultiOutputs | 18           | __MultiOutputs_14 |                          |                | description: MultiOutputsNode |
+----+--------------+--------------+-------------------+--------------------------+----------------+-------------------------------+
| 18 | Start        |              | __Start_18        |                          |                | description: Start            |
+----+--------------+--------------+-------------------+--------------------------+----------------+-------------------------------+
| 11 | Project      | 10           | __Project_11      |                          |                | description: Project          |
+----+--------------+--------------+-------------------+--------------------------+----------------+-------------------------------+
| 10 | GetNeighbors | 9            | __GetNeighbors_10 |                          |                | description: GetNeighbors     |
+----+--------------+--------------+-------------------+--------------------------+----------------+-------------------------------+
|  9 | Loop         | 14           | __Loop_9          |                          |                | description: Loop             |
+----+--------------+--------------+-------------------+--------------------------+----------------+-------------------------------+
|  8 | Project      | 7            | __UNAMED_VAR_2    | branch: true, node_id: 9 |                | description: Project          |
+----+--------------+--------------+-------------------+--------------------------+----------------+-------------------------------+
|  7 | GetNeighbors | 6            | __GetNeighbors_7  |                          |                | description: GetNeighbors     |
+----+--------------+--------------+-------------------+--------------------------+----------------+-------------------------------+
|  6 | Start        |              | __Start_6         |                          |                | description: Start            |
+----+--------------+--------------+-------------------+--------------------------+----------------+-------------------------------+

Sat, 01 Aug 2020 11:59:51 CST
(user@nebula) [nba]> profile go 3 steps from "Boris Diaw" over like yield like._dst union go 4 steps from "Tim Duncan" over like yield like._dst
+-------------------+
| like._dst         |
+-------------------+
| LaMarcus Aldridge |
+-------------------+
| Manu Ginobili     |
+-------------------+
| Tim Duncan        |
+-------------------+
| Tony Parker       |
+-------------------+
time spent 6655/7494 us
+----+--------------+--------------+-------------------+--------------------------+-----------------------------------------------------------+-------------------------------+
| id | name         | dependencies | output_var        | branch_info              | profiling_data                                            | description                   |
+----+--------------+--------------+-------------------+--------------------------+-----------------------------------------------------------+-------------------------------+
| 17 | DataCollect  | 13           | __DataCollect_17  |                          | version: 0, rows: 0, exec_time: 18us, total_time: 29us    | description: DataCollect      |
+----+--------------+--------------+-------------------+--------------------------+-----------------------------------------------------------+-------------------------------+
| 13 | Dedup        | 12           | __Dedup_13        |                          | version: 0, rows: 0, exec_time: 36us, total_time: 39us    | description: Dedup            |
+----+--------------+--------------+-------------------+--------------------------+-----------------------------------------------------------+-------------------------------+
| 12 | Union        | 5,11         | __Union_12        |                          | version: 0, rows: 0, exec_time: 29us, total_time: 32us    | description: Union            |
+----+--------------+--------------+-------------------+--------------------------+-----------------------------------------------------------+-------------------------------+
|  5 | Project      | 4            | __Project_5       |                          | version: 0, rows: 8, exec_time: 72us, total_time: 78us    | description: Project          |
+----+--------------+--------------+-------------------+--------------------------+-----------------------------------------------------------+-------------------------------+
|  4 | GetNeighbors | 3            | __GetNeighbors_4  |                          | version: 0, rows: 0, exec_time: 127us, total_time: 1248us | description: GetNeighbors     |
+----+--------------+--------------+-------------------+--------------------------+-----------------------------------------------------------+-------------------------------+
|  3 | Loop         | 14           | __Loop_3          |                          | version: 0, rows: 0, exec_time: 39us, total_time: 46us    | description: Loop             |
|    |              |              |                   |                          | version: 1, rows: 0, exec_time: 24us, total_time: 27us    |                               |
|    |              |              |                   |                          | version: 2, rows: 0, exec_time: 23us, total_time: 27us    |                               |
+----+--------------+--------------+-------------------+--------------------------+-----------------------------------------------------------+-------------------------------+
|  2 | Project      | 1            | __UNAMED_VAR_0    | branch: true, node_id: 3 | version: 0, rows: 2, exec_time: 58us, total_time: 64us    | description: Project          |
|    |              |              |                   |                          | version: 1, rows: 5, exec_time: 77us, total_time: 83us    |                               |
+----+--------------+--------------+-------------------+--------------------------+-----------------------------------------------------------+-------------------------------+
|  1 | GetNeighbors | 0            | __GetNeighbors_1  |                          | version: 0, rows: 0, exec_time: 137us, total_time: 1304us | description: GetNeighbors     |
|    |              |              |                   |                          | version: 1, rows: 0, exec_time: 123us, total_time: 1085us |                               |
+----+--------------+--------------+-------------------+--------------------------+-----------------------------------------------------------+-------------------------------+
|  0 | Start        |              | __Start_0         |                          | version: 0, rows: 0, exec_time: 2us, total_time: 33us     | description: Start            |
|    |              |              |                   |                          | version: 1, rows: 0, exec_time: 1us, total_time: 21us     |                               |
+----+--------------+--------------+-------------------+--------------------------+-----------------------------------------------------------+-------------------------------+
| 14 | MultiOutputs | 18           | __MultiOutputs_14 |                          | version: 0, rows: 0, exec_time: 0us, total_time: 5us      | description: MultiOutputsNode |
+----+--------------+--------------+-------------------+--------------------------+-----------------------------------------------------------+-------------------------------+
| 18 | Start        |              | __Start_18        |                          | version: 0, rows: 0, exec_time: 4us, total_time: 41us     | description: Start            |
+----+--------------+--------------+-------------------+--------------------------+-----------------------------------------------------------+-------------------------------+
| 13 | Dedup        | 12           | __Dedup_13        |                          | version: 0, rows: 0, exec_time: 36us, total_time: 39us    | description: Dedup            |
+----+--------------+--------------+-------------------+--------------------------+-----------------------------------------------------------+-------------------------------+
| 12 | Union        | 5,11         | __Union_12        |                          | version: 0, rows: 0, exec_time: 29us, total_time: 32us    | description: Union            |
+----+--------------+--------------+-------------------+--------------------------+-----------------------------------------------------------+-------------------------------+
|  5 | Project      | 4            | __Project_5       |                          | version: 0, rows: 8, exec_time: 72us, total_time: 78us    | description: Project          |
+----+--------------+--------------+-------------------+--------------------------+-----------------------------------------------------------+-------------------------------+
|  4 | GetNeighbors | 3            | __GetNeighbors_4  |                          | version: 0, rows: 0, exec_time: 127us, total_time: 1248us | description: GetNeighbors     |
+----+--------------+--------------+-------------------+--------------------------+-----------------------------------------------------------+-------------------------------+
|  3 | Loop         | 14           | __Loop_3          |                          | version: 0, rows: 0, exec_time: 39us, total_time: 46us    | description: Loop             |
|    |              |              |                   |                          | version: 1, rows: 0, exec_time: 24us, total_time: 27us    |                               |
|    |              |              |                   |                          | version: 2, rows: 0, exec_time: 23us, total_time: 27us    |                               |
+----+--------------+--------------+-------------------+--------------------------+-----------------------------------------------------------+-------------------------------+
|  2 | Project      | 1            | __UNAMED_VAR_0    | branch: true, node_id: 3 | version: 0, rows: 2, exec_time: 58us, total_time: 64us    | description: Project          |
|    |              |              |                   |                          | version: 1, rows: 5, exec_time: 77us, total_time: 83us    |                               |
+----+--------------+--------------+-------------------+--------------------------+-----------------------------------------------------------+-------------------------------+
|  1 | GetNeighbors | 0            | __GetNeighbors_1  |                          | version: 0, rows: 0, exec_time: 137us, total_time: 1304us | description: GetNeighbors     |
|    |              |              |                   |                          | version: 1, rows: 0, exec_time: 123us, total_time: 1085us |                               |
+----+--------------+--------------+-------------------+--------------------------+-----------------------------------------------------------+-------------------------------+
|  0 | Start        |              | __Start_0         |                          | version: 0, rows: 0, exec_time: 2us, total_time: 33us     | description: Start            |
|    |              |              |                   |                          | version: 1, rows: 0, exec_time: 1us, total_time: 21us     |                               |
+----+--------------+--------------+-------------------+--------------------------+-----------------------------------------------------------+-------------------------------+
| 14 | MultiOutputs | 18           | __MultiOutputs_14 |                          | version: 0, rows: 0, exec_time: 0us, total_time: 5us      | description: MultiOutputsNode |
+----+--------------+--------------+-------------------+--------------------------+-----------------------------------------------------------+-------------------------------+
| 18 | Start        |              | __Start_18        |                          | version: 0, rows: 0, exec_time: 4us, total_time: 41us     | description: Start            |
+----+--------------+--------------+-------------------+--------------------------+-----------------------------------------------------------+-------------------------------+
| 11 | Project      | 10           | __Project_11      |                          | version: 0, rows: 6, exec_time: 61us, total_time: 66us    | description: Project          |
+----+--------------+--------------+-------------------+--------------------------+-----------------------------------------------------------+-------------------------------+
| 10 | GetNeighbors | 9            | __GetNeighbors_10 |                          | version: 0, rows: 0, exec_time: 117us, total_time: 1148us | description: GetNeighbors     |
+----+--------------+--------------+-------------------+--------------------------+-----------------------------------------------------------+-------------------------------+
|  9 | Loop         | 14           | __Loop_9          |                          | version: 0, rows: 0, exec_time: 27us, total_time: 31us    | description: Loop             |
|    |              |              |                   |                          | version: 1, rows: 0, exec_time: 32us, total_time: 37us    |                               |
|    |              |              |                   |                          | version: 2, rows: 0, exec_time: 24us, total_time: 28us    |                               |
|    |              |              |                   |                          | version: 3, rows: 0, exec_time: 23us, total_time: 27us    |                               |
+----+--------------+--------------+-------------------+--------------------------+-----------------------------------------------------------+-------------------------------+
|  8 | Project      | 7            | __UNAMED_VAR_2    | branch: true, node_id: 9 | version: 0, rows: 2, exec_time: 65us, total_time: 74us    | description: Project          |
|    |              |              |                   |                          | version: 1, rows: 4, exec_time: 78us, total_time: 84us    |                               |
|    |              |              |                   |                          | version: 2, rows: 5, exec_time: 65us, total_time: 71us    |                               |
+----+--------------+--------------+-------------------+--------------------------+-----------------------------------------------------------+-------------------------------+
|  7 | GetNeighbors | 6            | __GetNeighbors_7  |                          | version: 0, rows: 0, exec_time: 148us, total_time: 1243us | description: GetNeighbors     |
|    |              |              |                   |                          | version: 1, rows: 0, exec_time: 111us, total_time: 1078us |                               |
|    |              |              |                   |                          | version: 2, rows: 0, exec_time: 142us, total_time: 1174us |                               |
+----+--------------+--------------+-------------------+--------------------------+-----------------------------------------------------------+-------------------------------+
|  6 | Start        |              | __Start_6         |                          | version: 0, rows: 0, exec_time: 1us, total_time: 25us     | description: Start            |
|    |              |              |                   |                          | version: 1, rows: 0, exec_time: 5us, total_time: 25us     |                               |
|    |              |              |                   |                          | version: 2, rows: 0, exec_time: 1us, total_time: 24us     |                               |
+----+--------------+--------------+-------------------+--------------------------+-----------------------------------------------------------+-------------------------------+

Sat, 01 Aug 2020 12:00:55 CST
(user@nebula) [nba]> explain format="dot" go 3 steps from "Boris Diaw" over like yield like._dst union go 4 steps from "Tim Duncan" over like yield like._dst
------------------------------------------------------
  plan
------------------------------------------------------
  digraph exec_plan {
      rankdir=LR;
      "DataCollect_17"[shape=box, style=rounded];
      "Dedup_13"->"DataCollect_17";
      "Dedup_13"[shape=box, style=rounded];
      "Union_12"->"Dedup_13";
      "Union_12"[shape=box, style=rounded];
      "Project_5"->"Union_12";
      "Project_11"->"Union_12";
      "Project_5"[shape=box, style=rounded];
      "GetNeighbors_4"->"Project_5";
      "GetNeighbors_4"[shape=box, style=rounded];
      "Loop_3"->"GetNeighbors_4";
      "Loop_3"[shape=diamond];
      "Project_2"->"Loop_3";
      "Loop_3"->"Start_0"[label="Do", style=dashed];
      "MultiOutputs_14"->"Loop_3";
      "Project_2"[shape=box, style=rounded];
      "GetNeighbors_1"->"Project_2";
      "GetNeighbors_1"[shape=box, style=rounded];
      "Start_0"->"GetNeighbors_1";
      "Start_0"[shape=box, style=rounded];
      "MultiOutputs_14"[shape=box, style=rounded];
      "Start_18"->"MultiOutputs_14";
      "Start_18"[shape=box, style=rounded];
      "Project_11"[shape=box, style=rounded];
      "GetNeighbors_10"->"Project_11";
      "GetNeighbors_10"[shape=box, style=rounded];
      "Loop_9"->"GetNeighbors_10";
      "Loop_9"[shape=diamond];
      "Project_8"->"Loop_9";
      "Loop_9"->"Start_6"[label="Do", style=dashed];
      "MultiOutputs_14"->"Loop_9";
      "Project_8"[shape=box, style=rounded];
      "GetNeighbors_7"->"Project_8";
      "GetNeighbors_7"[shape=box, style=rounded];
      "Start_6"->"GetNeighbors_7";
      "Start_6"[shape=box, style=rounded];
  }
------------------------------------------------------

Sat, 01 Aug 2020 12:03:47 CST

(user@nebula) [nba]> explain format="dot:struct" go 3 steps from "Boris Diaw" over like yield like._dst union go 4 steps from "Tim Duncan" over like yield like._dst
--------------------------------------------------------
  plan
--------------------------------------------------------
  digraph exec_plan {
      rankdir=LR;
      "DataCollect_17"[shape=box, style=rounded];
      "Dedup_13"->"DataCollect_17";
      "Dedup_13"[shape=box, style=rounded];
      "Union_12"->"Dedup_13";
      "Union_12"[shape=box, style=rounded];
      "Project_5"->"Union_12";
      "Project_11"->"Union_12";
      "Project_5"[shape=box, style=rounded];
      "GetNeighbors_4"->"Project_5";
      "GetNeighbors_4"[shape=box, style=rounded];
      "Loop_3"->"GetNeighbors_4";
      "Loop_3"[shape=diamond];
      "MultiOutputs_14"->"Loop_3";
      "Project_2"[shape=box, style=rounded];
      "GetNeighbors_1"->"Project_2";
      "Project_2"->"Loop_3"[label="Do", style=dashed];
      "GetNeighbors_1"[shape=box, style=rounded];
      "Start_0"->"GetNeighbors_1";
      "Start_0"[shape=box, style=rounded];
      "MultiOutputs_14"[shape=box, style=rounded];
      "Start_18"->"MultiOutputs_14";
      "Start_18"[shape=box, style=rounded];
      "Project_11"[shape=box, style=rounded];
      "GetNeighbors_10"->"Project_11";
      "GetNeighbors_10"[shape=box, style=rounded];
      "Loop_9"->"GetNeighbors_10";
      "Loop_9"[shape=diamond];
      "MultiOutputs_14"->"Loop_9";
      "Project_8"[shape=box, style=rounded];
      "GetNeighbors_7"->"Project_8";
      "Project_8"->"Loop_9"[label="Do", style=dashed];
      "GetNeighbors_7"[shape=box, style=rounded];
      "Start_6"->"GetNeighbors_7";
      "Start_6"[shape=box, style=rounded];
  }
--------------------------------------------------------

Sat, 01 Aug 2020 12:04:19 CST
```
## execution plan

image

### CODE REVIEW

File: printer/printDataSet.go [hunk 1]
```diff
@@ -4,32 +4,27 @@
...
 	"strconv"
-	"strings"
 
-	common "github.com/vesoft-inc/nebula-go/v2/nebula"
+	"github.com/jedib0t/go-pretty/v6/table"
+	"github.com/vesoft-inc/nebula-go/v2/nebula"
 )
 
-func val2String(value *common.Value, depth uint) string {
-	// TODO(shylock) get golang runtime limit
```
Comment (Shylock-Hg) [inline printer/printDataSet.go:19]: Why remove this , will lead stack over.

File: go.mod [hunk 2]
```diff
@@ -1,7 +1,13 @@
-module vesoft-inc/nebula-console
+module github.com/vesoft-inc/nebula-console
+
+go 1.13
 
 require (
-	github.com/vesoft-inc/nebula-go/v2 v2.0.0-20200722013035-73eebb836e7e
+	github.com/chzyer/logex v1.1.10 // indirect
+	github.com/chzyer/test v0.0.0-20180213035817-a1ea475d72b1 // indirect
+	github.com/jedib0t/go-pretty/v6 v6.0.4
```
Comment (Shylock-Hg) [inline go.mod:8]: I still don't think use the 3rd-party personal library is a good choice.

File: printer/printDataSet.go [hunk 3]
```diff
@@ -40,7 +35,7 @@ func val2String(value *common.Value, depth uint) string {
 	} else if value.IsSetFVal() { // float64
 		return strconv.FormatFloat(value.GetFVal(), 'g', -1, 64)
 	} else if value.IsSetSVal() { // string
-		return "\"" + string(value.GetSVal()) + "\""
```
Comment (Shylock-Hg) [inline printer/printDataSet.go:43]: If you remove "" there will be some mistake about `1` or `"1"` and many other cases.

File: main.go [hunk 4]
```diff
@@ -57,19 +58,23 @@ func printResp(resp *graph.ExecutionResponse, duration time.Duration) {
 		return
 	}
 	// Show table
-	if resp.GetData() != nil {
-		t.PrintTable(resp.GetData())
+	if resp.IsSetData() {
+		printer.PrintDataSet(resp.GetData())
+		// Show time
+		fmt.Printf("time spent %d/%d us", resp.GetLatencyInUs(), duration /*ns*/ /1000)
```
Comment (dutor) [inline main.go:64]: Remove `/*ns*/`, it is obvious to be inferred.

File: main.go [hunk 5]
```diff
@@ -94,12 +99,11 @@ func loop(client *ngdb.GraphClient, c Cli) error {
 			log.Fatalf("Execute error, %s", err.Error())
 		}
 		printResp(resp, duration)
-		fmt.Println(time.Now().Format("2006-01-02 15:04:05"))
+		// fmt.Println(time.Now().Format("2006-01-02 15:04:05"))
```
Comment (dutor) [inline main.go:102]: What's this for? Remove it if not needed.

Comment (Shylock-Hg) [reply main.go:102]: It's feature from 1.0

File: printer/printDataSet.go (hunk 1 above)
Comment (yixinglu) [reply printer/printDataSet.go]: Restore

File: printer/printDataSet.go (hunk 3 above)
Comment (yixinglu) [reply printer/printDataSet.go]: It's too noisy for response with many string columns.

File: README.md [hunk 6]
```diff
@@ -4,7 +4,12 @@ The console for Nebula Graph 2.0
 
 # Build
 
-Install golang by https://golang.org/doc/install, then try `go build`
+Install golang by https://golang.org/doc/install, then build by following commands:
```
[the remark this answers is not in the document]
Comment (yixinglu) [reply README.md]: fixed it.

File: main.go (hunk 4 above)
Comment (yixinglu) [reply main.go]: fixed it.

File: Makefile [hunk 7]
```diff
@@ -0,0 +1,16 @@
+
+.PHONY: build vendorbuild clean fmt
+
+default: build
+
+build: clean fmt
+	go build -o nebula-console;
```
[the remark this answers is not in the document]
Comment (yixinglu) [reply Makefile]: fixed it.

File: main.go (hunk 5 above)
Comment (yixinglu) [reply main.go]: Fixed it.

File: go.mod (hunk 2 above)
Comment (yixinglu) [reply go.mod:8]: There are many corner cases which need to be considered, like newline or tab in column. And we should not spend too much energy on this aspect.

File: cli/cli.go [hunk 8]
```diff
@@ -14,111 +14,33 @@ import (
...
-	// (user@nebula) [(space)] >
+	// (user@nebula) [space] >
 	if isTTY {
 		prompt += fmt.Sprintf("%s%s%s", ttyColorPrefix, ttyColorBold, ttyColorSuffix)
 	}
 	if isTTY && isErr {
 		prompt += fmt.Sprintf("%s%s%s", ttyColorPrefix, ttyColorRed, ttyColorSuffix)
 	}
-	prompt += fmt.Sprintf("(%s@%s) [(%s)]> ", user, NebulaLabel, space)
+	spaceName := "(none)"
```
[the remark this answers is not in the document]
Comment (yixinglu) [reply cli/cli.go:39]: Here is same as version 1.0

File: cli/cli.go [hunk 9]
```diff
@@ -14,111 +14,33 @@ import (
...
+	if len(space) > 0 {
+		spaceName = space
+	}
```
[the remark this answers is not in the document]
Comment (yixinglu) [reply cli/cli.go:42]: use `(none)` to differentiate from the real space named `none`

File: go.mod (hunk 2 above)
Comment (Shylock-Hg) [inline go.mod:8]: Does it support multi-line in one table cell?

File: .editorconfig [hunk 10]
```diff
@@ -0,0 +1,12 @@
+root = true
```
Comment (Shylock-Hg) [inline .editorconfig:1]: Ignore the editor configuration is better

File: go.mod (hunk 2 above)
Comment (yixinglu) [reply go.mod:8]: See above examples.

File: .editorconfig (hunk 10 above)
Comment (yixinglu) [reply .editorconfig:1]: ? Add it into .gitignore file?

Comment (Shylock-Hg) [reply .editorconfig:1]: Yes, it seems not the configuration about go project.

Comment (yixinglu) [reply .editorconfig:1]: No, this config will control the file render way for github repo

**Pull request merged**